src/model/SubSurface.hpp

```diff
@@ -263,6 +263,12 @@ namespace model {
 
     boost::optional<AirflowNetworkSurface> airflowNetworkSurface() const;
 
+    /* Get the total area of the sub surface including frame and divider area */
```

Comment:
```suggestion
    /* Get the total area of the sub surface rough area which includes the frame */
```

The divider doesn't extend outside of the glazed area.